x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/SecurityRestFilter.java

```diff
@@ -88,6 +90,11 @@ public void handleRequest(RestRequest request, RestChannel channel, NodeClient c
                         e -> handleException("Secondary authentication", request, channel, e)));
                 }, e -> handleException("Authentication", request, channel, e)));
         } else {
+            if (request.method() != Method.OPTIONS) {
+                HeaderWarning.addWarning("Elasticsearch built-in security features are not enabled, your cluster may be accessible " +
+                    "without authentication. Read https://www.elastic.co/guide/en/elasticsearch/reference/" + Version.CURRENT.major + "." +
+                    Version.CURRENT.minor + "/get-started-enable-security.html for more information");
```

Comment:
```suggestion
                containsString("Elasticsearch built-in security features are not enabled. Without authentication, your cluster could be accessible " +
                    "to anyone. See https://www.elastic.co/guide/en/elasticsearch/reference/" + Version.CURRENT.major + "." +
                    Version.CURRENT.minor + "/security-minimal-setup.html to enable security."));
```

Reply:
I updated the link with this suggested change and also modified the language a bit. Let me know if those changes are 👍